### ISSUE TITLE
[BUG] EVM chain priorityFeeGwei, need to get from block explorer

### ISSUE THREAD

**Issue by johnnyluo** [voltix-vault/voltix#140]
Currently , priorityFeeGwei has been hardcoded as 0 , which doesn't work at the moment ,  because it return at least 1 as priority fee based on etherscan
<img width="1087" alt="image" src="https://github.com/voltix-vault/voltix/assets/749608/63cbcb36-60fa-46ff-a1ff-989ed1c632c1">

We need to get the priorityFeeGwei from etherscan

based on their api
```
{
   "status":"1",
   "message":"OK",
   "result":{
      "LastBlock":"13053741",
      "SafeGasPrice":"20",
      "ProposeGasPrice":"22",
      "FastGasPrice":"24",
      "suggestBaseFee":"19.230609716",
      "gasUsedRatio":"0.370119078777807,0.8954731,0.550911766666667,0.212457033333333,0.552463633333333"
   }
}
```
`priorityFeeGwei = ProposeGasPrice - suggestBaseFee`

**Comment by enriquesouza** [comment]
tks.

**Comment by johnnyluo** [comment]
fixed in PR #143
